UnityProject/Assets/Scripts/Items/Medical/DefibrillatorPaddles.cs

```diff
@@ -1,128 +1,157 @@
+				{
+					livingHealthMaster.playerScript.Mind.OrNull()?.StopGhosting();
+					_ = SoundManager.PlayNetworkedAtPosAsync(soundSuccsuess, objectPos);
+					StartCoroutine(Cooldown());
+					return;
+				}
+
 				_ = SoundManager.PlayNetworkedAtPosAsync(soundFailed, objectPos);
 				StartCoroutine(Cooldown());
-				return;
 			}
-			livingHealthMaster.RestartHeart();
-			_ = SoundManager.PlayNetworkedAtPosAsync(soundZap, objectPos);
-			if (livingHealthMaster.IsDead == false)
+
+			if (isReady == false || onCooldown == true)
 			{
-				livingHealthMaster.playerScript.Mind.OrNull()?.StopGhosting();
-				_ = SoundManager.PlayNetworkedAtPosAsync(soundSuccsuess, objectPos);
-				StartCoroutine(Cooldown());
+				Chat.AddExamineMsg(interaction.Performer,
+					$"You need to charge the {gameObject.ExpensiveName()} first!");
 				return;
 			}
-			_ = SoundManager.PlayNetworkedAtPosAsync(soundFailed, objectPos);
-			StartCoroutine(Cooldown());
+
+			var bar = StandardProgressAction.Create(
+				new StandardProgressActionConfig(StandardProgressActionType.CPR, false, false, true), Perform);
+			bar.ServerStartProgress(interaction.Performer.RegisterTile(), Time, interaction.Performer);
 		}
 
-		if (isReady == false || onCooldown == true)
+		private IEnumerator Cooldown()
 		{
-			Chat.AddExamineMsg(interaction.Performer, $"You need to charge the {gameObject.ExpensiveName()} first!");
-			return;
+			onCooldown = true;
+			yield return WaitFor.Seconds(cooldown);
+			onCooldown = false;
+			SoundManager.PlayNetworkedAtPos(soundCharged, gameObject.AssumedWorldPosServer());
 		}
-		var bar = StandardProgressAction.Create(new StandardProgressActionConfig(StandardProgressActionType.CPR, false, false, true), Perform);
-		bar.ServerStartProgress(interaction.Performer.RegisterTile(), Time, interaction.Performer);
-	}
-
-	private IEnumerator Cooldown()
-	{
-		onCooldown = true;
-		yield return WaitFor.Seconds(cooldown);
-		onCooldown = false;
-		SoundManager.PlayNetworkedAtPos(soundCharged, gameObject.AssumedWorldPosServer());
-	}
 
-	public void ServerPerformInteraction(HandActivate interaction)
-	{
-		if (onCooldown)
+		public void ServerPerformInteraction(HandActivate interaction)
 		{
-			Chat.AddExamineMsg(interaction.Performer, $"The {gameObject.ExpensiveName()} is still charging!");
-			return;
+			if (onCooldown)
+			{
+				Chat.AddExamineMsg(interaction.Performer, $"The {gameObject.ExpensiveName()} is still charging!");
+				return;
+			}
+
+			if (isReady == false)
+			{
+				Chat.AddExamineMsg(interaction.Performer, $"You prepare the {gameObject.ExpensiveName()}");
+				isReady = true;
+				_ = SoundManager.PlayNetworkedAtPosAsync(soundReady, gameObject.AssumedWorldPosServer());
+				return;
+			}
+
+			Chat.AddExamineMsg(interaction.Performer,
+				$"<color=green>The {gameObject.ExpensiveName()} is charged and ready to be used.</color>");
 		}
-		if (isReady == false)
+
+		public void OnDropOrThrow(GameObject droppedObject)
```

Comment:
ok,  there's a few byond items that have a funny behaviour defibrillator  paddles and cyborg items I think it might be worth making a standard component to Simplify it?